stellargraph/layer/hinsage.py

```diff
@@ -581,6 +581,13 @@ def in_out_tensors(self):
             model output tensor(s) of shape (batch_size, layer_sizes[-1]).
 
         """
+
+        if multiplicity is not None and multiplicity != self.multiplicity:
```

Comment:
just curious, why can't we set a different `multiplicity` to `self.multiplicity` here.

Reply:
Ah, this is unnecessary now. My original thought here was that this was the simplest way to have DGI support HinSAGE, given it passes in `multiplicity=1`, and it could be generalised in future. In particular, there's various support structures (like sampling-schema) that depend on the multiplicity and so would need to be recomputed if it was customised here.

However, #1243 was extended since this was written to have `DeepGraphInfomax` no longer pass a `multiplicity` argument, so I'll remove this. Good catch.